modules/lang-painless/src/main/antlr/PainlessLexer.g4

```diff
@@ -25,7 +25,7 @@ protected abstract boolean isSlashRegex();
 }
 
 WS: [ \t\n\r]+ -> skip;
-COMMENT: ( '//' .*? [\n\r] | '/*' .*? '*/' ) -> skip;
+COMMENT: ( '//' .*? ([\n\r] | EOF) | '/*' .*? '*/' ) -> skip;
```

Comment:
@camerondurham this is just a question, do you think we could simplify it to something like:

```
COMMENT: ( '//' ~[\r\n]* | '/*' .*? '*/' ) -> skip;
```

`EOF` does work but it looks like super edge case

Reply:
Yes, I think your definition makes more sense instead of coding for the edge case. From a quick test in [ANTLR LAB](http://lab.antlr.org/) also seems this works with the edge cases I tried. Rebased regenerated the lexer Java with this change.

Thanks!